sympy/physics/continuum_mechanics/beam.py

```diff
@@ -210,6 +213,24 @@ def reaction_loads(self):
         """ Returns the reaction forces in a dictionary."""
         return self._reaction_loads
 
+    @property
+    def rotation_jumps(self):
+        """
+        Returns the value for the rotation jumps in rotation hinges in a dictionary.
+        The rotation jump is the rotation (in degrees) in a rotation hinge. This can
```

Comment:
My guess is that this would be radians, not degrees. The angle computed from a slope with atan() gives radians.

Reply:
You are right, I changed it